lucien-ios/AddAndEditNewComicBook/ComicFormViewController.swift

```diff
@@ -348,7 +413,7 @@ extension AddComicViewController: UITextFieldDelegate {
     }
 }
 
-extension AddComicViewController: CameraViewControllerDelegate {
+extension ComicFormViewController: CameraViewControllerDelegate {
 
     // MARK: - CameraViewControllerDelegate
 
```

Comment:
In the delegate function below, the `updateCoverButton` function you created should be called so there's no code duplication